dmutils/cloudfoundry.py

```diff
@@ -3,15 +3,12 @@
 """
 
 import json
+import os
+from typing import Union
 
 
-def get_vcap_services(app, default=None) -> dict:
-    if app.config.get("VCAP_SERVICES") is None:
-        return default
-
-    vcap_services = json.loads(app.config["VCAP_SERVICES"])
-
-    return vcap_services
+def get_vcap_services() -> Union[dict, KeyError]:
```

Comment:
```suggestion
def get_vcap_services() -> dict]:
```
I think this return type can just be `dict` It might raise a `keyError` instead, but if it does return something, that something will be a dict